.github/workflows/on-release.yml

```diff
@@ -62,7 +50,6 @@ jobs:
             echo "The current branch is not strictly ahead of the release branch. Cannot finish transaction without touching release branch history."
             exit 1
           fi
-
       - name: Create or update release branch
         run: git push origin "${WORKING_BRANCH}":"${RELEASE_BRANCH}"
```

Comment:
WORKING_BRANCH isn't set anymore, is it?

Reply:
done